.github/workflows/build.yaml

```diff
@@ -0,0 +1,37 @@
+name: build
+
+on:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          bazel build //:for_all_platforms
+          mv bazel-out/darwin_arm64-opt/bin/external/libarchive\~/tar/bsdtar tar_darwin_arm64
```

Comment:
is there some symmetric delete of these lines?

Reply:
i don't think so, they are all different :(

Reply:
I mean, where's the method we were using before?

Reply:
are you asking how this was working before? https://github.com/aspect-build/bsdtar-prebuilt/blob/fa28210938d7c069194ec92a34643c31eaaeb9c2/.github/workflows/release.yml#L12-L19